src/components/vuestic-components/va-topbar/VaTopbarLink.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <router-link
```

Comment:
`li` is not ideal. Should be `a`. Reasons:
* target does nothing for `li`.
* Can't `Ctrl + Click` to open in new page.
* Can't copy url.

Also notice this kinda makes some `ul`'s lose value, so we might want to swap those with `div`'s

Reply:
fixed